firmware/firmware.ino

```diff
@@ -19,7 +19,10 @@
 
 #include <Arduino.h>
 #include <Wire.h>
+#if defined(ARDUINO_ARCH_AVR) || defined(ARDUINO_ARCH_ESP32) || defined(ARDUINO_ARCH_ESP8266)
+#define USE_EEPROM
 #include <EEPROM.h>
+#endif
```

Comment:
Disable EEPROM unless on known good architectures. The `#define` means we don't have to repeat the exact `#if` expression elsewhere.

Reply:
ok